code/modules/mob/living/carbon/carbon.dm

```diff
@@ -978,13 +978,25 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 		if(do_after(src, breakouttime, 0, target = src))
 			if(I.loc != src || buckled)
 				return
-			visible_message("<span class='danger'>[src] manages to remove [I]!</span>")
-			to_chat(src, "<span class='notice'>You successfully remove [I].</span>")
+			if(istype(I, /obj/item/restraints/handcuffs/cable/twimsts))
+				visible_message("<span class='danger'>[src] manages to eat through [I]!</span>")
+				to_chat(src, "<span class='notice'>You successfully eat through [I].</span>")
+			else
+				visible_message("<span class='danger'>[src] manages to remove [I]!</span>")
+				to_chat(src, "<span class='notice'>You successfully remove [I].</span>")
```

Comment:
You could move this ```to_chat``` into the ```visible_message```.

![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/ab83a7d6-f1a1-4440-9562-ab7087751464)